### PR TITLE
moving security from status-im namespace to root level utils directory

### LINKED ISSUE #14551: move utils.security-html and utils.security from status-im to status-im2
I need this for an existing PR : https://github.com/status-im/status-mobile/pull/14514
and I believe moving this to new namespace would result in me updating its old references to new namespace as well.
All of this will be a lot of file changes in one PR and hence I have created this issue for that purpose.

### PR DESCRIPTION
**Pull request by siddarthkay**
security related files below are moved from : 

`src/status_im/utils/security.cljs` to `src/utils/security/core.cljs`
`src/status_im/utils/security_html.cljs` to `src/utils/security/security_html.cljs`
`src/status_im/utils/security_html_test.cljs` to `src/utils/security/security_html_test.cljs`
`src/status_im/utils/security_test.cljs` to `src/utils/security/security_test.cljs`

I have updated all usage of security from old namespace to the new one.

fixes #14551

status: ready

### CODE REVIEW

File: doc/decisions/0007-masking-sensitive-data.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ printed out by mistake in a log entry (see https://github.com/status-im/status-m
 
 To minimize the risk of leaking passwords through logs, we should not pass
 passwords as strings in our codebase. We introduced a new type `MaskedData` in
-`status-im.utils.security`.
+`utils.security.core`.
```
Comment (flexsurfer) [inline doc/decisions/0007-masking-sensitive-data.md]: i don't think we should change outdated files

Comment (flexsurfer) [reply doc/decisions/0007-masking-sensitive-data.md]: probably better to rollback this change

Comment (siddarthkay) [reply doc/decisions/0007-masking-sensitive-data.md]: hmmm.. do you think its worth adding a comment over there in the md file stating `status-im.utils.security` is now `utils.security.core` ?

Comment (siddarthkay) [reply doc/decisions/0007-masking-sensitive-data.md]: my point here being no one should be confused when they find `status-im.utils.security` anywhere in codebase..

**Review by J-Son89 — APPROVED** [review]

**Review by ilmotta — APPROVED** [review]

File: doc/decisions/0007-masking-sensitive-data.md (hunk 1 above)
Comment (flexsurfer) [reply doc/decisions/0007-masking-sensitive-data.md]: ok lets keep it, thanks

**Comment by siddarthkay** [comment]
@qoqobolo @pavloburykh @churik : Hello Lovely QA Team, This PR migrates majority of the security related codebase from an old namespace to a new namespace. This should ideally change 0 functionality but it may or may not introduce bugs, hence I would request manual-qa for this one :)

**Comment by qoqobolo** [comment]
@siddarthkay thanks for your work! 
No issues were found and the failed tests are not related to PR.
Ready for merge.

**Comment by siddarthkay** [comment]
> @siddarthkay thanks for your work! No issues were found and the failed tests are not related to PR. Ready for merge.

Thanks @qoqobolo !
I'll rebase and merge :)

**Pull request merged**